src/Parser/VectorTileParser.js

```diff
@@ -3,12 +3,29 @@ import Protobuf from 'pbf';
 import { VectorTile } from '@mapbox/vector-tile';
 import { worldDimension3857 } from 'Core/Geographic/Extent';
 import { FeatureCollection, FEATURE_TYPES } from 'Core/Feature';
+import { featureFilter } from '@mapbox/mapbox-gl-style-spec';
+import Style from 'Core/Style';
 
 const VectorTileFeature = { types: ['Unknown', 'Point', 'LineString', 'Polygon'] };
 const globalExtent = new Vector3(worldDimension3857.x, worldDimension3857.y, 1);
 const lastPoint = new Vector2();
 const firstPoint = new Vector2();
-
+const styleCache = new Map();
+
+function parseStyle(vtFeature, layer) {
```

Comment:
Quick link as a comment to the specification of a VectorTile style ? Thanks

Reply:
done